django_project/lesson/views/section.py

```diff
@@ -126,7 +132,329 @@ def get_queryset(self):
                     'The requested project does not exist.'
                 )
             section_qs = section_qs.filter(
-                project=project).order_by('-section_number')
+                project=project).order_by('section_number')
             return section_qs
         else:
             raise Http404('Sorry! We could not find your section!')
+
+
+class SectionDetailView(SectionMixin, DetailView):
+    """Detail view for Section."""
+
+    context_object_name = 'section'
+    template_name = 'section/detail.html'
+
+    def get_context_data(self, **kwargs):
+        """Get the context data which is passed to a template.
+
+        :param kwargs: Any arguments to pass to the superclass.
+        :type kwargs: dict
+
+        :returns: Context data which will be passed to the template.
+        :rtype: dict
+        """
+        context = super(SectionDetailView, self).get_context_data(**kwargs)
+
+        return context
+
+    def get_queryset(self):
+        """Get the queryset for this view.
+
+        :returns: Queryset for All Sections
+        :rtype: QuerySet
+        """
+
+        qs = Section.objects.all()
```

Comment:
This is already the default, no? Is the view not based on the model already?

Reply:
Ah yes. Thanks.